kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/kube_apiserver_metrics.py

```diff
@@ -16,14 +16,27 @@
     'go_goroutines': 'go_goroutines',
     'APIServiceRegistrationController_depth': 'APIServiceRegistrationController_depth',
     'etcd_object_counts': 'etcd_object_counts',
+
+    # For Kubernetes < 1.14
     'rest_client_request_latency_seconds': 'rest_client_request_latency_seconds',
     'apiserver_admission_webhook_admission_latencies_seconds': 'admission_webhook_admission_latencies_seconds',
     'apiserver_admission_step_admission_latencies_seconds': 'admission_step_admission_latencies_seconds',
-    'apiserver_admission_controller_admission_duration_seconds': 'admission_controller_admission_duration_seconds',
+    'apiserver_admission_controller_admission_latencies_seconds': 'admission_controller_admission_duration_seconds',
     # fmt: off
     'apiserver_admission_step_admission_latencies_seconds_summary':
         'admission_step_admission_latencies_seconds_summary',
     # fmt: on
+
+    # For Kubernetes >= 1.14
+    # (https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.14.md#deprecated-metrics)
+    'rest_client_request_duration_seconds': 'rest_client_request_latency_seconds',
+    'apiserver_admission_webhook_duration_latencies_seconds': 'admission_webhook_admission_latencies_seconds',
```

Comment:
should this be `apiserver_admission_webhook_duration_seconds` ?

Reply:
It should be `apiserver_admission_webhook_admission_duration_seconds` !
Thanks for spotting this typo, @celenechang !!
This is now fixed.